src/services/rfqm_service.ts

```diff
@@ -73,6 +74,7 @@ export interface FetchIndicativeQuoteResponse {
 
 export interface FetchFirmQuoteParams {
     apiKey: string;
```

Comment:
might be worth leaving a TODO on this field (`apiKey`) as well to remove in the future